### DISCUSSION TITLE
setNumOfHttpDatabaseThreadsHint doesnt work?

### DISCUSSION THREAD

**Discussion by Robert-Earth** [pelicanmapping/osgearth#1982] [Support Q&A] [answered]
@gwaldron 
Sorry for bothering you. I want to setup multi threads to download tile images. No matter how many threads be set, it is still only two threads in network, the maximum speed less than 150K,  although my computer has 24 cores.  is there anything wrong like this: 

osg::DisplaySettings::instance()->setNumOfDatabaseThreadsHint( 16 );
osg::DisplaySettings::instance()->setNumOfHttpDatabaseThreadsHint( 8 );

or,
 int cores = Registry::capabilities().getNumProcessors();
    osg::DisplaySettings::instance()->setNumOfDatabaseThreadsHint( osg::clampAbove(cores, 2) );
    osg::DisplaySettings::instance()->setNumOfHttpDatabaseThreadsHint( osg::clampAbove(cores/2, 1) );

**Comment by gwaldron** [comment, marked as answer]
Please paste the output of `osgearth_version --caps`.
